src/server/game/Instances/InstanceScript.h

```diff
@@ -21,6 +21,7 @@
 
 #include "ZoneScript.h"
 #include "Common.h"
+#include "ObjectMgr.h"
```

Comment:
Hrm, just use the actual type (`std::unordered_multimap<` etc) instead of the typedef?